packages/cli/src/controllers/folder.controller.ts

```diff
@@ -21,9 +21,9 @@ import { FolderNotFoundError } from '@/errors/folder-not-found.error';
 import { BadRequestError } from '@/errors/response-errors/bad-request.error';
 import { InternalServerError } from '@/errors/response-errors/internal-server.error';
 import { NotFoundError } from '@/errors/response-errors/not-found.error';
-import type { ListQuery } from '@/requests';
 import { AuthenticatedRequest } from '@/requests';
 import { FolderService } from '@/services/folder.service';
+import type { ListQuery } from '@/types-db';
```

Comment:
not in this PR, but we should definitely migrate all these older `class-validator` stuff over to DTOs in the `api-types` package.